enterprise/backend/src/metabase_enterprise/metabot_v3/tools/filters.clj

```diff
@@ -182,14 +182,20 @@
 (defn- add-aggregation
   [query aggregation]
   (let [expr     (bucketed-column aggregation)
+        sort-order (:sort-order aggregation)
         agg-expr (case (:function aggregation)
                    :count          (lib/count)
                    :count-distinct (lib/distinct expr)
                    :sum            (lib/sum expr)
                    :min            (lib/min expr)
                    :max            (lib/max expr)
-                   :avg            (lib/avg expr))]
-    (lib/aggregate query agg-expr)))
+                   :avg            (lib/avg expr))
+        query-with-aggregation (lib/aggregate query agg-expr)]
+    (if sort-order
+      (let [query-aggregations (lib/aggregations query-with-aggregation)
+            last-aggregation-idx (dec (count query-aggregations))]
+        (lib/order-by query-with-aggregation (lib/aggregation-ref query-with-aggregation last-aggregation-idx) sort-order))
```

Comment:
This is probably not the best way to implement it. Basically if there is a sort order defined on the aggregation, this adds it to the query. I'm not sure if there is a better way to grab the aggregation-ref from the aggregation we've just added to the query